lib/activerecord-import/base.rb

```diff
@@ -13,6 +13,7 @@ def self.base_adapter(adapter)
     when 'postgresql_makara' then 'postgresql'
     when 'makara_postgis' then 'postgresql'
     when 'postgis' then 'postgresql'
+    when 'cockroachdb' then 'postgresql'
```

Comment:
Looks like this needs to be added here too: https://github.com/zdennis/activerecord-import/blob/80bda221af3a9a478ae2c14c0edde5b210defa3e/Rakefile#L16

Reply:
@willie if you can make this small update I'll get this merged. Thanks!

Reply:
Actually, let's hold off on that for now.